README.md

```diff
@@ -152,6 +152,7 @@ suggesting data producers include the Attributes used by MetGenC in their netCDF
 | geospatial_lon_max (global)   |                | R                | |
 | geospatial_lat_min (global)   |                | R                | |
 | geospatial_lat_max (global)   |                | R                | |
+| geospatial_bounds (global)    |                |                  | |
```

Comment:
Should be the same as `geospatial_lat_max` etc.: `R` under `NSIDC Guidelines`

Reply:
K, I wasn't sure if it was.